openedx/core/djangoapps/waffle_utils/__init__.py

```diff
@@ -107,12 +107,12 @@ def is_enabled(self, course_key=None):  # pylint: disable=arguments-differ
                 outside the context of any course.
         """
         if course_key:
-            assert isinstance(
-                course_key, CourseKey
-            ), "Provided course_key '{}' is not instance of CourseKey.".format(
-                course_key
-            )
-        is_enabled_for_course = self._get_course_override_value(course_key)
-        if is_enabled_for_course is not None:
-            return is_enabled_for_course
+            if isinstance(course_key, CourseKey):
+                is_enabled_for_course = self._get_course_override_value(course_key)
+                if is_enabled_for_course is not None:
+                    return is_enabled_for_course
+            else:
+                # In case this gets called with a content library key, that's fine - just ignore it and
+                # act like a normal waffle flag. We currently don't support library-specific overrides.
+                assert isinstance(course_key, LearningContextKey)
```

Comment:
Should we keep the explanation for the assertion? 
```
            [...] "Provided course_key '{}' is not instance of CourseKey.".format(
                course_key
            )
````

Reply:
Should also update the `is_enabled` method docstring so we know it's ok to call this with a LearningContextKey | CourseKey.

Reply:
Done.